app/queries/facility_yearly_follow_ups_and_registrations_query.rb

```diff
@@ -47,11 +47,9 @@ def year(month_date)
   def totals(yearly_data)
     count_columns = Reports::FacilityMonthlyFollowUpAndRegistration.column_names - NON_COUNT_FIELDS.map(&:to_s)
     yearly_data.each_with_object(Hash.new(0)) do |monthly_data, totals|
-      monthly_data.attributes.map do |key, value|
-        if count_columns.include?(key)
-          key = "yearly" + key[7..]
-          totals[key] += value
-        end
+      count_columns.map do |key|
+        new_key = key.gsub("monthly", "yearly")
```

Comment:
This looks great. We could ensure that the substitution happens only in the beginning of the key with a regex

`/^monthly/`, what do you think?

Reply:
yeah, that makes sense.